.travis/php/script.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+set -ev
+
```

Comment:
@mbohal @adamkudrna TwigYard contains configured Robo. Don't we want to use Robo instead of those tasks? If yes, we will not have to maintain same logic in two different files.

Reply:
I would prefer to run tests with `vendor/bin/robo test`. As it is implemented now it would be an easy mistake to make to only modify the tests run in one place and forget the other. In fact it already happened since travis is presently not running the phpstan test.

Reply:
Again, no problem to change it. It was there before and I just moved it to shell script.